src/html/shared/overlay.html

```diff
@@ -0,0 +1,21 @@
+<div class="c-overlay">
```

Comment:
I wouldn't recommend naming it `c-overlay` because you may get confused with this component being used as an overlay for your modal component later on.

Would suggest `c-menu`